pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -1013,16 +1014,61 @@ func (c *Reconciler) createRun(ctx context.Context, runName string, params []v1b
 	return c.PipelineClientSet.TektonV1alpha1().Runs(pr.Namespace).Create(ctx, r, metav1.CreateOptions{})
 }
 
-func getTaskrunWorkspaces(pr *v1beta1.PipelineRun, rpt *resources.ResolvedPipelineTask) ([]v1beta1.WorkspaceBinding, string, error) {
+func propagateWorkspaces(rpt *resources.ResolvedPipelineTask) (*resources.ResolvedPipelineTask, error) {
+	workspacesUsedInSteps := []string{}
```

Comment:
can this be a set? it'll make it easier to check "contains" in line 1036 below

Reply:
Sure.